app/pages/home-not-logged-in.cjsx

```diff
@@ -32,30 +32,31 @@ module.exports = React.createClass
             <source type="video/mp4" src="./assets/home-video.mp4"></source>
           </video>
         }
-
-        <h1>THE ZO<ZooniverseLogo />NIVERSE</h1>
-        <h2 className="lighter">is people powered research</h2>
-        <h2>
-          <Link to="/about" className="standard-button">Learn More</Link>
-        </h2>
+        <div className="home-intro-content">
+          <h1>THE ZO<ZooniverseLogo />NIVERSE</h1>
+          <h2 className="lighter">is people powered-research</h2>
+          <div className="home-intro-buttons">
+            <button className="button standard-button" onClick={@scrollDown}>Learn More</button>
+            <Link to="/projects" className="button intro-button">Get Started</Link>
+          </div>
+        </div>
+        <button className="home-link-down" onClick={@scrollDown}>
```

Comment:
Moving this into the section element is causing some weird partial underneath the section.